runtime/darwinia/src/pallets/society.rs

```diff
@@ -38,7 +38,7 @@ impl Config for Runtime {
 	type RotationPeriod = RotationPeriod;
 	type MaxLockDuration = MaxLockDuration;
 	type FounderSetOrigin = EnsureRootOrMoreThanHalfCouncil;
-	type SuspensionJudgementOrigin = EnsureFounder<Runtime>;
+	type SuspensionJudgementOrigin = EnsureFounder<Self>;
```

Comment:
![image](https://user-images.githubusercontent.com/11801722/157635530-5726bf90-f568-49cb-b0c7-edd4827ba079.png)

Do we need to update `type Randomness`?